_posts/2023-10-26-socratic-seminar-129.md

```diff
@@ -0,0 +1,265 @@
+- [Full Disclosure: CVE-2023-40231 / CVE-2023-40232 / CVE-2023-40233 / CVE-2023-40234 "All your mempool are belong to us"](https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-October/004122.html)
+- [Batch exchange withdrawal to lightning requires covenants](https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-October/004126.html)
+- [Removing channel reserve for mobile wallet users](https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-October/004136.html)
+- [On solving pinning, replacement cycling and mempool issues for bitcoin second-layers](https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-October/004158.html)
+
+
+#### [dlc-dev](https://mailmanlists.org/pipermail/dlc-dev)
+
+
+### Meetings
+- [Bitcoin PR Review Club](https://bitcoincore.reviews)
+    - [#28107 Type-safe transaction identifiers](https://bitcoincore.reviews/28107)
+    - [#28368 Fee Estimator updates from Validation Interface/CScheduler thread](https://bitcoincore.reviews/28368)
+
+- Bitcoin Core general developer meetings
+    - [September 28](https://www.erisian.com.au/bitcoin-core-dev/log-2023-09-28.html#l-193)
+    - [October 5](https://www.erisian.com.au/bitcoin-core-dev/log-2023-10-05.html#l-186)
+    - [October 12](https://www.erisian.com.au/bitcoin-core-dev/log-2023-10-12.html#l-123)
+    - [October 17](https://www.erisian.com.au/bitcoin-core-dev/log-2023-10-17.html#l-100)
+    - [October 19](https://www.erisian.com.au/bitcoin-core-dev/log-2023-10-19.html#l-187)
+
+- Lightning Specification meeting
+    - <!--- TODO replace: [December 5th](https://github.com/lightning/bolts/issues/1046) --->
+- Core Lightning Developer Call
+    - <!--- TODO replace: [September 20th](https://diyhpl.us/wiki/transcripts/c-lightning/2021-09-20-developer-call/) --->
+- dlc-specs meetings
+    - <!--- TODO replace: [October 5th](https://github.com/discreetlogcontracts/dlcspecs/pull/175) --->
+- Lightning specification meetings
+    - <!--- TODO replace: [October 11th](https://github.com/lightningnetwork/lightning-rfc/issues/920) --->
+- Bitcoin Contracting Primitives Working Group
+	- <!--- TODO replace: [April 18th](https://github.com/ariard/bitcoin-contracting-primitives-wg/blob/main/meetings/meetings-18-04.md) --->
+
+### Optech
+- [Bitcoin Optech Newsletter #274](https://bitcoinops.org/en/newsletters/2023/10/25/)
+- [Bitcoin Optech Newsletter #273](https://bitcoinops.org/en/newsletters/2023/10/18/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/19/)
+- [Bitcoin Optech Newsletter #272](https://bitcoinops.org/en/newsletters/2023/10/11/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/12/)
+- [Bitcoin Optech Newsletter #271](https://bitcoinops.org/en/newsletters/2023/10/04/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/05/)
+- [Bitcoin Optech Newsletter #270](https://bitcoinops.org/en/newsletters/2023/09/27/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/09/28/)
```

Comment:
It’s a bit odd that we order e.g. the developer meetings oldest first, but the Newsletters youngest first.

```suggestion
- [Bitcoin Optech Newsletter #270](https://bitcoinops.org/en/newsletters/2023/09/27/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/09/28/)
- [Bitcoin Optech Newsletter #271](https://bitcoinops.org/en/newsletters/2023/10/04/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/05/)
- [Bitcoin Optech Newsletter #272](https://bitcoinops.org/en/newsletters/2023/10/11/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/12/)
- [Bitcoin Optech Newsletter #273](https://bitcoinops.org/en/newsletters/2023/10/18/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/19/)
- [Bitcoin Optech Newsletter #274](https://bitcoinops.org/en/newsletters/2023/10/25/)
```

Reply:
yeah and PRs are listed top down from most recent merge to oldest. I can flip some of the lists in scraper so there's consistency